src/_data/sidenav.yml

```
@@ -102,6 +102,11 @@
           permalink: /ui/animations/hero-animations
         - title: Staggered animations
           permalink: /ui/animations/staggered-animations
+    - title: Scrolling
+      permalink: /ui/scrolling
+      children:
```

If you want your new introduction page to show up in the sidenav, you'll need to add it here with whatever name you'd like.

```suggestion
      children:
        - title: Introduction
          permalink: /ui/scrolling
          match-page-url-exactly: true
```